redshift_connector/cursor.py

```diff
@@ -444,10 +474,12 @@ def write_dataframe(self: "Cursor", df: "pandas.DataFrame", table: str) -> None:
         try:
             import pandas
         except ModuleNotFoundError:
-            raise ModuleNotFoundError(MISSING_MODULE_ERROR_MSG.format(module="pandas"))
+            raise ModuleNotFoundError(
+                MISSING_MODULE_ERROR_MSG.format(module="pandas"))
```

Comment:
@Yash621 is this necessary? adds noise to the PR.

Reply:
fixed in revisions